samza-core/src/main/scala/org/apache/samza/job/local/ProcessJobFactory.scala

```diff
@@ -87,13 +106,13 @@ class ProcessJobFactory extends StreamJobFactory with Logging {
     val commandBuilder = ReflectionUtil.getObj(commandBuilderClass, classOf[CommandBuilder])
 
     // JobCoordinator is stopped by ProcessJob when it exits
```

Comment:
Could you please update this comment? It looks like it was out-of-date before you made your changes, but would be nice to clarify it now.